### ISSUE TITLE
Links not working

### ISSUE THREAD

**Issue by BrunoMarshall** [TechFlow-Space/Raffle-frontend#2]
![image](https://user-images.githubusercontent.com/35432495/91018718-37d77400-e5f0-11ea-92aa-fe2a0acbccbe.png)
The links are currently not working

**Comment by sophoah** [comment]
the app is missing an history of the purchase, hope the history link is that it is supposed to be :)

**Comment by TechFlowSpace** [comment]
Yeah, its a work in progress. We will include feature to view past 7 days history.
